InternetRamen.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@InternetRamen"
+project: Handwired w/ 3d hotswappable 3by2 Macro Pad
+---
+
+# Project Name
+Handwired w/ 3d hotswappable 3by2 Macro Pad
+
+## Summary
+I'm going to build a 6 key macro pad. This is basically a mini keyboard. Super excited for this build because it's my first project with hardware other than my PC. I've also wanted to build a keyboard from scratch and this is very similar, but managable.
+## Plan
+
+I'm going to build this using [this](https://www.youtube.com/watch?v=v9r5DKZLz68) guide. It does not require soldering and uses hand wiring to achieve the process.
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| JLCPCB 3D Printing   | https://jlcpcb.com/ | $21.94  |
```

Comment:
You're still pretty far under the budget limit. What if you got a 3D printer and printed these pieces yourself?